File: LeetCode/Easy/3330-Find_The_Original_Typed_String_I/3330-FindTheOriginalTypedStringI.js
/**
 * @param {string} word
 * @return {number}
 */
var possibleStringCount = function (word) {
    let count = 1;

    for (let i = 1; i < word.length; i++) {
        if (word[i - 1] == word[i])
            count++;
    }

    return count
};

File: LeetCode/Easy/3330-Find_The_Original_Typed_String_I/3330-FindTheOriginalTypedStringI.py
class Solution:
    def possibleStringCount(self, word: str) -> int:
        count = 1
        prev = word[0]

        for i in range(1, len(word)):
            if word[i] == prev:
                count += 1

            prev = word[i]

        return count


a = Solution()
print(a.possibleStringCount(word="abbcccc"))
print(a.possibleStringCount(word="abcd"))
print(a.possibleStringCount(word="aaaa"))
